### PR TITLE
Community page

### LINKED ISSUE #7: Add the Community Page
## Goal
The goal of this issue is to recreate the Community Page https://emberjs.com/community/ in an Ember app. The currently deployed site is a Ruby app found [in a different repository](https://github.com/emberjs/website), and we are working towards converting the whole site over.

## Criteria
- [ ] Add route: /community
- [ ] The content look should be the same as the live site
- [ ] use ember-styleguide components (and make notes where there is not an appropriate component)

## Additional Info
- To get started with this issue, please add a comment that you are interested in helping out with it. 
- If you have questions, need to get someone's attention for feedback, or it's your first time contributing, the learning team can help you out! The best way to get in touch is to join the [Ember Community Discord](https://discord.gg/emberjs) and join the #dev-ember-learning channel. Thank you!

### PR DESCRIPTION
**Pull request by abhilashlr**
Fixes #7

### CODE REVIEW

File: app/styles/global/_sections.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+section {
+  margin: 3rem 0;
```
Comment (sivakumar-kailasam) [inline app/styles/global/_sections.scss]: I don't think you need this anymore. This must already be handled by ember styleguide

**Comment by abhilashlr** [comment]
@mansona, this branch seems to render the UI fine for me.

**Comment by mansona** [comment]
@abhilashlr if you go to https://deploy-preview-94--ember-beta-website.netlify.com/community/ you will see what I mean

Also, it could be browser specific 🤔 what browser are you testing this in?

**Comment by abhilashlr** [comment]
> @abhilashlr if you go to https://deploy-preview-94--ember-beta-website.netlify.com/community/ you will see what I mean
> 
> Also, it could be browser specific 🤔 what browser are you testing this in?

Hey @mansona, looks like some styles are not available from the ember-styleguide anymore. I think, something has changed for `section` CSS. I will have a look.

**Pull request merged**